assets/src/components/resource/editors/ContentEditor.tsx

```diff
@@ -57,11 +57,11 @@ interface ContentOutlineItemProps extends OutlineItemProps {
 
 export const ContentOutlineItem = (props: ContentOutlineItemProps) => {
   const { contentItem } = props;
-
+  const title = contentItem.editor === 'markdown' ? 'Markdown' : 'Paragraph';
```

Comment:
The outline in the left will list "Markdown" instead of "Paragraph" now, but maybe even "Paragraph" should be changed to "Rich text" or something since it can represent much more than a paragraph.